src/lib/components/layout/Sidebar.svelte

```diff
@@ -346,195 +346,185 @@
 			{/if}
 
 			<div class="pl-2 my-2 flex-1 flex flex-col space-y-1 overflow-y-auto">
-				{#each $chats.filter((chat) => {
-					if (search === '') {
-						return true;
-					} else {
-						let title = chat.title.toLowerCase();
-						const query = search.toLowerCase();
-
-						let contentMatches = false;
-						// Access the messages within chat.chat.messages
-						if (chat.chat && chat.chat.messages && Array.isArray(chat.chat.messages)) {
-							contentMatches = chat.chat.messages.some((message) => {
-								// Check if message.content exists and includes the search query
-								return message.content && message.content.toLowerCase().includes(query);
-							});
-						}
-
-						return title.includes(query) || contentMatches;
-					}
-				}) as chat, i}
-					<div class=" w-full pr-2 relative">
-						{#if chatTitleEditId === chat.id}
-							<div
-								class=" w-full flex justify-between rounded-xl px-3 py-2 hover:bg-gray-900 {chat.id ===
-								$chatId
-									? 'bg-gray-900'
-									: ''} transition whitespace-nowrap text-ellipsis"
-							>
-								<input bind:value={chatTitle} class=" bg-transparent w-full outline-none mr-10" />
-							</div>
-						{:else}
-							<a
-								class=" w-full flex justify-between rounded-xl px-3 py-2 hover:bg-gray-900 {chat.id ===
-								$chatId
-									? 'bg-gray-900'
-									: ''} transition whitespace-nowrap text-ellipsis"
-								href="/c/{chat.id}"
-							>
-								<div class=" flex self-center flex-1 w-full">
-									<div
-										class=" text-left self-center overflow-hidden {chat.id === $chatId
-											? 'w-[160px]'
-											: 'w-full'} "
-									>
-										{chat.title}
-									</div>
+				{#each $chats as chat, i}
```

Comment:
Thanks for the PR, I'm not sure if this approach is more readable in this specific case.

Reply:
should i revert the sidebar one?